tarea04

rclbox opened the pull request:
tenemos que calcular el promedio de tres valores, de tres tipos de datos
diferentes, estos valores son ingresados desde el teclado para que el
usuario pueda poner los 3 valores que desee
version 0.1

rclbox committed:
tarea04

tenemos que calcular el promedio de tres valores, de tres tipos de datos
diferentes, estos valores son ingresados desde el teclado para que el
usuario pueda poner los 3 valores que desee

cvhupci commented on hw_01_id_14.cpp:
```diff
@@ -1,19 +1,26 @@
+    a= (k+l+m)/3;//hacemos la operacion para calcular el promedio 
+    b= (o+p+q)/3;
+	c= (x+w+z)/3;
+    cout<<"el primer promedio es\t"<<a<<endl;//imprimimos el valor del promedio
```
deberia de respetar el tipo de dato de los números  enteros, e imprimir el resultado como entero no como double, esto lo puede hacer haciendo un casting

cvhupci commented on hw_01_id_14.cpp:
```diff
@@ -1,19 +1,26 @@
+	cin>> x; cin>>w; cin>>z;
+    a= (k+l+m)/3;//hacemos la operacion para calcular el promedio 
+    b= (o+p+q)/3;
+	c= (x+w+z)/3;
```
deberia de respetar el tipo de dato char, e imprimir el resultado como char no como double, esto lo puede hacer haciendo un casting

rclbox replied:
Bueno solo le agregue a cada tipo de dato las variables de cada promedio para que fueran del mismo tipo

rclbox committed:
homework

con cin ponemos tres tipos de datos con el teclado de tipo entero,
double y char y calculamos su promedio con su mis po tipo de dato
respectivamente

cvhupci commented on hw_01_id_14.cpp:
```diff
@@ -1,19 +1,26 @@
+	c= (x+w+z)/3;
+    cout<<"el primer promedio es\t"<< a<<endl;//imprimimos el valor del promedio
+    cout<<"el segundo promedio es\t"<<b<<endl;
+    cout<<"el tercer promedio es\t"<< c <<endl;
```
.... y la diferencia esta en que ahora el promedio de las variables de tipo caracter no los imprime como numericos sino como caracter
